src/__tests__/InputField.js

```diff
@@ -24,6 +24,105 @@ describe('InputField', () => {
     )
     console.error.mockRestore()
   })
+  test('it renders using FormField alias', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Input id="with-left-icon" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select ', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Select id="with-no-options" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select and Icon', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Select id="with-left-icon" />
```

Comment:
The other thought is that maybe the `id` of the `Select` can decouple with the context to the `Icon` and just stay simple with "with-no-options", like the `id` values in other unit tests.